data-processing/explanations/colorize_tex.py

```diff
@@ -145,7 +146,8 @@ def colorize_citations(tex: str) -> Iterator[CitationColorizationBatch]:
             hue_generator = generate_hues()
 
             # And get the hue for the next entity.
-            hue = next(hue_generator)
+            if hue is None:
```

Comment:
Feeling like a Python 3 newbie here. Can you verify for me that hue won't be 'None' when the generator yields StopIterator? I'm pretty sure that's not the case, but I just want to rule that potential bug.

Reply:
nice catch; that would be a bug, check latest push